vet.sh

```diff
@@ -120,12 +114,25 @@ if [[ -z "${VET_SKIP_PROTO}" ]]; then
     (git status; git --no-pager diff; exit 1)
 fi
 
-# - Check that our modules are tidy.
-if go help mod >& /dev/null; then
-  find . -name 'go.mod' | xargs -IXXX bash -c 'cd $(dirname XXX); go mod tidy'
+# - gofmt, goimports, golint (with exceptions for generated code), go vet,
+# go mod tidy.
+# Perform these checks on each module inside gRPC.
+for MOD_FILE in $(find . -name 'go.mod'); do
+  MOD_DIR=$(dirname ${MOD_FILE})
+  if [[ ${MOD_DIR} == "./test/tools" ]]; then
+    continue
+  fi
```

Comment:
Why do we need to skip this?

Reply:
Would you mind removing the build constraint?  That predates go modules, and is no longer needed.

Reply:
I am a bit confused about the use of this `test/tools` module. If we remove `// +build tools`, then those imports will cause errors(detected by `go vet`) such as 
```
import "github.com/client9/misspell/cmd/misspell" is a program, not an importable package
```.
And if we remove those imports, `go mod tidy` will complain about the dependency. I am out of my way. Do you have any idea on how to solve this?

Reply:
Two options:

1. import any other package in `github.com/client9/misspell` instead.
2. Add a mostly-empty `.go` file in `test/tools` with no build tag so `vet` doesn't freak out.

(2) is probably easiest

Reply:
Done with add an empty `.go` file  in `test/tools`. Could you please take a look at it again? Thank you so much!